webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.js

```diff
@@ -13,54 +13,59 @@ const PageLayout = ({
   searchProps,
   searchQuery,
   onSearch,
+  onBookmarkClick,
   customBreadcrumbs,
   breadcrumbOptions,
   toolbarButtons,
   toastNotifications,
   children,
-}) => (
-  <div id="main">
-    <div id="react-content">
-      {toastNotifications && (
-        <div
-          id="toast-notifications-container"
-          data-notifications={toastNotifications}
-        >
-          <ToastsList />
-        </div>
-      )}
-      <div id="breadcrumb">
-        {!breadcrumbOptions && (
-          <div className="row form-group">
-            <h1 className="col-md-8">{header}</h1>
+}) => {
+  document.title = header;
```

Comment:
Can we create a small helper function for this operation (e.g. `updateDocumentTitle(header)`)?
IMO it can live here: https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/document.js